pkg/features/kube_features.go

```diff
@@ -1102,7 +1103,7 @@ var defaultKubernetesFeatureGates = map[featuregate.Feature]featuregate.FeatureS
 
 	PodDeletionCost: {Default: true, PreRelease: featuregate.Beta},
 
-	PodDisruptionConditions: {Default: true, PreRelease: featuregate.Beta},
+	PodDisruptionConditions: {Default: true, PreRelease: featuregate.GA, LockToDefault: true}, // remove in 1.33
```

Comment:
please clean up the file test/e2e/nodefeature/nodefeature.go - delete the feature mention there.

Reply:
Makes sense, I also open the PR to clean up the obsolete configuration: https://github.com/kubernetes/test-infra/pull/32868